cmd/historygen/main.go

```diff
@@ -5,16 +5,22 @@ import (
 	"log"
 	"time"
 
-	"github.com/jjneely/stuff/tsdb"
+	"github.com/qcrisw/stuff/tsdb"
```

Comment:
This wont work merging back into my repository from your fork.

Reply:
Yeah sure, I had no other choice to test it on my fork, and was actually waiting for this comment 😆 . Will send a commit shortly.